src/plugins/plugin.ts

```diff
@@ -5,13 +5,13 @@
  * repository.
  */
 
-import { Documentalist } from "..";
+import { ICompiler } from "../compiler";
```

Comment:
should this just be defined in this file? then `compiler.ts` implements this interface.